src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033004> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000192> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0012419>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033005> (serous secreting cell of bronchus submucosal gland)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:5487122"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:9651178"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033005> "A(n) serous secreting cell that is part of a(n) bronchus submucosal gland."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033005> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033005> "serous secreting cell of bronchus submucosal gland"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033005> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000313> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410043>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033006> (endothelial cell of lymphatic vessel of efferent lymphatic vessel)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32094869"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033006> "A(n) endothelial cell of lymphatic vessel that is part of a(n) efferent lymphatic vessel."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033006> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033006> "endothelial cell of lymphatic vessel of efferent lymphatic vessel"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033006> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0002138> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0010397>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033007> (brush cell of bronchus of epithelium of lobar bronchus)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:15817800"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033007> "A(n) brush cell of bronchus that is part of a(n) epithelium of lobar bronchus."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033007> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033007> "brush cell of bronchus of epithelium of lobar bronchus"^^xsd:string)
```

Comment:
Consider using the label 'brush cell of epithelium of lobar bronchus' or simply 'brush cell of lobar bronchus'.

Reply:
In this case, we can automatically generate the label 'brush cell of epithelium of lobar bronchus'  by specifying 'brush cell' + 'epithelium of lobar bronchus' in the table.  From inspecting the Equivalent Class axioms in this branch of CL it is clear that some normalization and correction is needed.  All should have brush cell as the genus & have some appropriately specific Uberon term in the differentia.